TechEmpower/TFB-Documentation
# Documentation cleanup

Makes a bunch of fixes to outdated or incorrect information in the docs, including the following:

1. Project Information
    1. Concepts
        1. Fixes a typo
    2. Framework_tests
        1. Clarifies the explanation of pre-rendered buffers for plaintext tests
    3. Expected-Questions
        1. Updated the answer about the 15s test time now that we are actively working on continuous benchmarking
        2. Updated the Amazon EC2 answer to be about cloud environments in general
        3. Removed a redundant description of outdated hardware
    4. Results-Website
        1. Fixed formatting of results FAQ to better match the formatting on the Expected-Questions page
2. Development
    1. Index
        1. Reworded the introduction paragraph
    2. Readme-Formats
        1. added links to examples of fairly robust READMEs
    3. Contributing-Guide
        1. Updated the Github Pull Request Procedure to match our new branching scheme
    4. Add-Benchmarking-Scripts
        1. Fixed a reference to install.sh
    5. Add-Frameworks-or-Tests
        1. Fixed a reference to install.sh
    6. Testing-and-Debugging
        1. fixed a typo and a broken link
3. Benchmarking
    1. Getting-Started-Benchmarking
        1. Remove separate setup process for Linux benchmarking servers and just redirect the user to the development setup
4. Codebase
    1. Index
        1. Updated directory structure
    2. Framework-Files
        1. Removed section about install.sh and moved pertinent information to the setup.sh section
    3. Setup-Files
        Remove mentions of bash_profile.sh
5. About
    1. Framework-Benchmarks
        1. fixed typo
